tools/llm_bench/task/audio_generation.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import os
+import time
+import numpy as np
+from pathlib import Path
+import hashlib
+import logging as log
+import llm_bench_utils
+import llm_bench_utils.model_utils as model_utils
+import llm_bench_utils.metrics_print as metrics_print
+import llm_bench_utils.gen_output_data as gen_output_data
+
+FW_UTILS = {'pt': llm_bench_utils.pt_utils, 'ov': llm_bench_utils.ov_utils}
+
+
+def streamer(word: str) -> bool:
+    print(word, end="")
+    return False
+
+
+def run_speech_2_txt_generation(raw_speech, pipe, args, num, md5_list, audio_id,
+                                iter_data_list, mem_consumption, processor):
+    result_md5_list = []
+    max_rss_mem_consumption = ''
+    max_uss_mem_consumption = ''
+    max_shared_mem_consumption = ''
+    if (args['mem_consumption'] == 1 and num == 0) or args['mem_consumption'] == 2:
+        mem_consumption.start_collect_memory_consumption()
+    start = time.perf_counter()
+    result_text = pipe.generate(
+        raw_speech,
+        max_new_tokens=1000,
+        # 'task' and 'language' parameters are supported for multilingual models only
+        language="<|en|>",
+        task="transcribe",
+        return_timestamps=True,
+        streamer=streamer,
```

Comment:
why do you need streamer? Does it will not give some messing in output logs printing during model execution? Usually we prefer wait whole output generated and then decode it for text generation. In some cases we used streamer, but not for output printing as alternative way to calculate inference token time